index.html

```diff
@@ -150,5 +150,28 @@ <h2 class="list-of-skills">Skills</h2>
       </div>
     </div>
   </section>
+  <section class="contact-section">
+    <div class="contact-details">
+      <h2 class="paragraph_1">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+    </div>
+    <form class="form" action="">
```

Comment:
:red_circle: You have done well implementing the form. But the action attribute is empty and the method attribute is non-existent. The **method** attribute specifies how to send form-data (the form-data is sent to the page specified in the **action** attribute). [The  project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md) state that you should use the [Formspree service (Click here for more)](https://formspree.io/html) to Collect data. 


![Screenshot from 2023-01-17 19-51-06](https://user-images.githubusercontent.com/26736582/212975092-c4f09254-1e01-40da-88d1-0c30c4b91c70.png)

The action value should be similar to `https://formspree.io/f/{form_id}` and method value equals POST